lnwallet/channel.go

```diff
@@ -2507,7 +2524,8 @@ func (lc *LightningChannel) createCommitmentTx(c *commitment,
 // reflects the current state of HTLCs within the remote or local commitment
 // chain.
 func (lc *LightningChannel) evaluateHTLCView(view *htlcView, ourBalance,
-	theirBalance *lnwire.MilliSatoshi, nextHeight uint64, remoteChain bool) *htlcView {
+	theirBalance *lnwire.MilliSatoshi, nextHeight uint64,
```

Comment:
Comment above should be updated to indicate the semantics of the `mutate` modifier.